internal/provider/vaultsecrets/resource_vault_secrets_rotating_secret.go

```diff
@@ -206,6 +213,29 @@ func (r *resourceVaultSecretsRotatingSecret) Schema(_ context.Context, _ resourc
 				exactlyOneRotatingSecretTypeFieldsValidator,
 			},
 		},
+		"azure_application_password_params": schema.SingleNestedAttribute{
```

Comment:
I'd remove the params suffix to align with the other credential types

Reply:
Agreed. Changed it.